scripts/parse_data_from_cldr.py

```diff
@@ -0,0 +1,143 @@
+def _parse_multiplier_words(number, word, language_data):
+    required_part = word.split("<")[-1]
+    if required_part[0] != " ":
+        return
+
+    power_of_10 = _find_zeroes(number)
+    if '$' in required_part:
+        valid_words = re.findall(CAPTURE_BRACKET_CONTENT, required_part)
+        for valid_word in valid_words:
+            power_of_10_num = pow(10, power_of_10)
+            if power_of_10 >= 2:
+                language_data["MULTIPLIERS"][valid_word] = power_of_10_num
+
+    else:
+        valid_word = required_part.split("[")[0].strip()
+        power_of_10_num = pow(10, power_of_10)
+        if power_of_10 >= 2:
+            language_data["MULTIPLIERS"][valid_word] = power_of_10_num
+
+
+def _extract_information(key, word, language_data):
+    try:
+        number = int(key)
+        word = word.replace(";", '')
+        count_greater_than_sign = word.count('>')
+        count_less_than_sign = word.count('<')
+        count_equal_to_sign = word.count('=')
+        if count_equal_to_sign != 0:
+            return
+
+        if count_greater_than_sign == 0 and count_less_than_sign == 0:
+            _parse_base_words(number, word, language_data)
+        elif count_greater_than_sign == 2 and count_less_than_sign == 0:
+            _parse_compound_words(number, word, language_data)
+        elif count_greater_than_sign == 2 and count_less_than_sign == 2:
+            _parse_multiplier_words(number, word, language_data)
+
+    except ValueError:
+        print("The given key {} is not an integer".format(key))
+
+
+def write_complete_data():
+    for files in os.listdir(SOURCE_PATH):
+        full_source_path = os.path.join(SOURCE_PATH, files)
+        full_target_path = os.path.join(TARGET_PATH, files.split(".")[0]+".py")
+        full_supplementary_path = os.path.join(SUPPLEMENTARY_PATH, files)
+
+        language_data_populated = {}
+
+        with open(full_source_path, 'r', encoding='utf-8') as source:
+            language_data = {}
+            for keys in REQUIRED_DATA_POINTS:
+                language_data[keys] = {}
+
+            data = json.load(source)
+            try:
+                requisite_data = data['rbnf']['rbnf']['SpelloutRules']
+            except KeyError:
+                print("This key doesn't exist in {}".format(files))
+                continue
+
+            for keys, vals in requisite_data.items():
+                if(_is_valid(keys)):
+                    for key, val in vals.items():
+                        _extract_information(key, val, language_data)
+
+            language_data_populated = language_data
+
+        with open(full_supplementary_path, 'r', encoding='utf-8') as supplement:
+            data = json.load(supplement)
+            for keys in REQUIRED_DATA_POINTS:
+                language_data_populated[keys].update(data[keys])
+
+        encoding_comment = "# -*- coding: utf-8 -*-\n"
+        translation_data = json.dumps(language_data_populated, indent=4, ensure_ascii=False)
```

Comment:
I think that it would be a really good idea to "order" each section by value.

So, instead of having 
```
   "UNIT_NUMBERS": {
        "cero": 0,
        "uno": 1,
        "dos": 2,
        "tres": 3,
      ...
        "un": 1,
        "una": 1
    },
```

We will have:

```
   "UNIT_NUMBERS": {
        "cero": 0,
        "uno": 1,
        "un": 1,
        "una": 1
        "dos": 2,
        "tres": 3,
      ...
    },
```
or similar.

This would make easier to read the files. I think that the easiest place to do it is here, after having all the `language_data`, but it's up to you.